From: Ricardo Salveti de Araujo <rsalveti@rsalveti.net>
Date: Tue, 29 Sep 2015 19:39:32 -0300
Subject: Tell GCC to ignore -Wdate-time (gcc 4.9)

Signed-off-by: Ricardo Salveti de Araujo <rsalveti@rsalveti.net>
---
 core/rtw_debug.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/core/rtw_debug.c b/core/rtw_debug.c
index af9303c..3555c10 100644
--- a/core/rtw_debug.c
+++ b/core/rtw_debug.c
@@ -62,8 +62,11 @@ u32 GlobalDebugLevel = _drv_err_;
 
 void dump_drv_version(void *sel)
 {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdate-time"
 	DBG_871X_SEL_NL(sel, "%s %s\n", DRV_NAME, DRIVERVERSION);
 	DBG_871X_SEL_NL(sel, "build time: %s %s\n", __DATE__, __TIME__);
+#pragma GCC diagnostic pop
 }
 
 void dump_log_level(void *sel)
