From: Ricardo Salveti de Araujo <rsalveti@rsalveti.net>
Date: Tue, 29 Sep 2015 20:01:03 -0300
Subject: autoconf.h: enable ioctl_cfg80211

Signed-off-by: Ricardo Salveti de Araujo <rsalveti@rsalveti.net>
---
 include/autoconf.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/autoconf.h b/include/autoconf.h
index fac9eac..d36a250 100755
--- a/include/autoconf.h
+++ b/include/autoconf.h
@@ -36,11 +36,11 @@
 #define PLATFORM_LINUX	1
 
 
-//#define CONFIG_IOCTL_CFG80211 1
+#define CONFIG_IOCTL_CFG80211 1
 
 #ifdef CONFIG_IOCTL_CFG80211
-	//#define RTW_USE_CFG80211_STA_EVENT /* Indecate new sta asoc through cfg80211_new_sta */
-	#define CONFIG_CFG80211_FORCE_COMPATIBLE_2_6_37_UNDER
+	#define RTW_USE_CFG80211_STA_EVENT /* Indecate new sta asoc through cfg80211_new_sta */
+	//#define CONFIG_CFG80211_FORCE_COMPATIBLE_2_6_37_UNDER
 	//#define CONFIG_DEBUG_CFG80211 
 	//#define CONFIG_DRV_ISSUE_PROV_REQ // IOT FOR S2
 	#define CONFIG_SET_SCAN_DENY_TIMER
